ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/sflow.j2

```diff
@@ -28,15 +34,28 @@ sflow vrf {{ vrf.name }} source-interface {{ vrf.source_interface }}
 {%         endif %}
 {{ destination_cli }}
 {%     endfor %}
+{%     if sflow.source is arista.avd.defined %}
```

Comment:
Are source and source_interface allowed at the same time in CLI? If not, use an `elif` and choose one as the winner. (Document that if both are configured, xxx will take precedence)

Reply:
you are right Claus, they are mutually exclusive